tests/playwright/page-object-model/trust/contacts-page.ts

```diff
@@ -1,39 +1,49 @@
 import { Locator, Page, expect } from '@playwright/test'
-import { MockTrustsProvider } from '../../mocks/mock-trusts-provider'
 import { TrustHeaderComponent } from '../shared/trust-header-component'
 import { TrustNavigationComponent } from '../shared/trust-navigation-component'
+import { FakeTestData } from '../../fake-data/fake-test-data'
 
 export class ContactsPage {
   readonly expect: ContactsPageAssertions
   readonly trustHeading: TrustHeaderComponent
   readonly trustNavigation: TrustNavigationComponent
   readonly pageHeadingLocator: Locator
 
-  constructor (readonly page: Page) {
+  fakeTestData: FakeTestData
+
+  constructor (readonly page: Page, fakeTestData: FakeTestData) {
+    this.fakeTestData = fakeTestData
     this.expect = new ContactsPageAssertions(this)
     this.trustHeading = new TrustHeaderComponent(page)
     this.trustNavigation = new TrustNavigationComponent(page)
     this.pageHeadingLocator = page.locator('h1')
   }
 
-  async goTo (
-    ukprn: string = MockTrustsProvider.expectedFormattedTrustResult.ukprn
-  ): Promise<void> {
-    await this.page.goto(`/trusts/contacts/${ukprn}`)
+  async goTo (): Promise<void> {
+    const uid = this.fakeTestData.getFirstTrust().uid
```

Comment:
We could have a field in this class for the currently selected trust and set it here. We could then use it elsewhere in the class and assertions

Reply:
This pattern should work for trust details too.

Reply:
There is an issue with this for the navigation spec, as we are not navigating to the page using 'goTo' - should we leave as is, or check for an undefined 'currentTrust' in 'toBeOnTheRightPage'?

Reply:
`toBeOnTheRightPage` (for details and contacts) is only used by the navigation tests. I think we should remove the trust check